config/base.yml

```diff
@@ -18,3 +18,9 @@ paths:
   sources:  '{{ paths.builds }}/sources'
   tars:     '{{ paths.builds }}/tars'
   logs:     '{{ paths.builds }}/logs'
+connect:
+  socket: '/tmp/docker.sock'
+  tcp:
+    force: false
```

Comment:
Im thinking of changing the approach and simply map the options passed to dockerode -- so if they change something then we wouldnt have to write anything magic on our side:

* https://github.com/apocas/dockerode#getting-started

ie:

``` yaml
connect:
  socketPath: ...

# or

connect:
  host: ...
  port: ...
```

but basically we can grab the configuration and just send it to dockerode. In case that config is missing, we can resort to `{socketPath: /tmp/docker.sock}`